examples/large_models/gpt_fast/handler.py

```diff
@@ -44,15 +44,23 @@ def __init__(self):
 
     def initialize(self, ctx):
         self.context = ctx
+        properties = self.context.system_properties
         rank = maybe_init_dist()
 
         self.local_rank = rank if rank is not None else 0
 
         if torch.cuda.is_available():
+            if (
+                "torchrun" not in ctx.model_yaml_config
+                and properties.get("gpu_id") is not None
+            ):
+                gpu_id = properties.get("gpu_id")
+            else:
+                gpu_id = self.local_rank
```

Comment:
this part is not necessary.

Reply:
Understood. Will update in subsequent commit.